spec/requests/relationship_operations_spec.rb

```diff
@@ -291,29 +291,33 @@
     context 'authorized for remove_to_many_relationship' do
       context 'not limited by policy scopes' do
         before do
-          allow_operations('remove_to_many_relationship', [
-            [article, comments_to_remove.first, :comments],
-            [article, comments_to_remove.second, :comments]
-          ])
+          allow_operation('remove_to_many_relationship', article, kind_of(Array), "comments")
         end
 
         it { is_expected.to be_successful }
       end
 
       context 'limited by policy scope on comments' do
-        let(:comments_scope) { Comment.none }
         before do
-          allow_operation('remove_to_many_relationship', article, kind_of(Comment), :comments)
+          allow_any_instance_of(ArticlePolicy).to receive(
+            :remove_from_comments?).and_return(true)
         end
 
-        it { is_expected.to be_not_found }
+        let(:comments_scope) { Comment.none }
+
+        it 'calls :remove_to_many_relationship with no comments in scope' do
+          expect_operation('remove_to_many_relationship', article, [], "comments")
+          last_response
+        end
+
+        it { is_expected.not_to be_not_found }
```

Comment:
Can we write this spec as either

```rb
it { is_expected.to be_successful }
```

or

```rb
it { is_expected.to be_forbidden }
```

like we have done with all the other specs?

If this feels a bit over your head, I could also tackle these at some point in the future.